src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.Llvm.cs

```diff
@@ -351,6 +362,14 @@ public static TypeDescriptor getTypeDescriptor(IntPtr thisHandle, CORINFO_CLASS_
             return typeDescriptor;
         }
 
+        [UnmanagedCallersOnly]
+        public static CORINFO_METHOD_STRUCT_* getCompilerHelpersMethodHandle(IntPtr thisHandle, byte* className, byte* methodName)
+        {
+            var _this = GetThis(thisHandle);
+
+            return _this.ObjectToHandle(_this._compilation.GetCompilerHelpersMethodDesc(Marshal.PtrToStringAnsi((IntPtr)className), Marshal.PtrToStringAnsi((IntPtr)methodName)));
```

Comment:
```suggestion
            return _this.ObjectToHandle(_this._compilation.GetCompilerHelpersMethodDesc(Marshal.PtrToStringUTF8((IntPtr)className), Marshal.PtrToStringUTF8((IntPtr)methodName)));
```
PtrToStringAnsi is legacy

Reply:
THanks, you did say.  Sorry.